### DISCUSSION TITLE
Installing ASReview on Mac Book Air

### DISCUSSION THREAD

**Discussion by mstoelti** [asreview/asreview#1721] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Hey:) 
i have a Mac Book Air from 2022 and wanted to install ASReview, but there is always this error: "command not found". 
But it's not for every command. Sometimes it works. 
I made a screenshot to see what's going on. 

If somebody has any idea, please let me now! 
![IMG_7573](https://github.com/asreview/asreview/assets/167449424/5c978f9b-6fb0-43cb-900c-d8bb2359937f)

Thank you!!

**Comment by Rensvandeschoot** [comment]
It seems it has to do with Python, not with ASReview. Have you tried the solution mentioned in the [troubleshooting](https://asreview.readthedocs.io/en/latest/troubleshooting.html#unknown-command-pip) section of the documentation?

**Comment by mstoelti** [comment]
Thanks for your qick response!

Yes i already tried all of those solutions. I also installed the Phython
again completely new, but it´s still not working so far.
Do you have any other ideas ?

Best,
Matilda

**Reply by jteijema** [reply]
What happens when you just type `python`? or `python3`? This issue is unrelated to ASReview.

`python --version`

**Comment by jteijema** [comment, marked as answer]
Issue is a likely duplicate of #1289 .

**Comment by mstoelti** [comment]
Yes it already worked. Thank you so much for the support!
